core/src/main/scala/org/apache/spark/status/api/v1/OneStageResource.scala

```diff
@@ -81,7 +83,8 @@ private[v1] class OneStageResource(ui: SparkUI) {
       @DefaultValue("20") @QueryParam("length") length: Int,
       @DefaultValue("ID") @QueryParam("sortBy") sortBy: TaskSorting): Seq[TaskData] = {
     withStageAttempt(stageId, stageAttemptId) { stage =>
-      val tasks = stage.ui.taskData.values.map{AllStagesResource.convertTaskData}.toIndexedSeq
+      val tasks = stage.ui.taskData.values.map{
```

Comment:
The style should be changed to `map {  AllStagesResource.convertTaskData(_, ui.lastUpdateTime) }`, requires whitespace between `{` and `}`. You could check other similar codes about the style.